cirq/contrib/noise_models/noise_models_test.py

```diff
@@ -157,3 +160,105 @@ def test_decay_noise_after_moment():
         cirq.measure(qubits[2], key='q2')
     ])
     _assert_equivalent_op_tree(true_noisy_program, noisy_circuit)
+
+
+# Fake calibration data for mock object.
+_CALIBRATION = {
+    'name':
+    'projects/foo/processors/fake_processor_name/calibrations/1579214873',
+    'timestamp': '2020-01-16T14:47:51Z',
+    'data': {
+        '@type':
+        'type.googleapis.com/cirq.google.api.v2.MetricsSnapshot',
+        'timestampMs':
+        '1579214873',
+        'metrics': [{
+            'name': 'xeb',
+            'targets': ['0_0', '0_1'],
+            'values': [{
+                'doubleVal': .9999
+            }]
+        }, {
+            'name': 'xeb',
+            'targets': ['0_0', '1_0'],
+            'values': [{
+                'doubleVal': .9998
+            }]
+        }, {
+            'name': 'single_qubit_rb_total_error',
+            'targets': ['q0_0'],
+            'values': [{
+                'doubleVal': .001
+            }]
+        }, {
+            'name': 'single_qubit_rb_total_error',
+            'targets': ['q0_1'],
+            'values': [{
+                'doubleVal': .002
+            }]
+        }, {
+            'name': 'single_qubit_rb_total_error',
+            'targets': ['q1_0'],
+            'values': [{
+                'doubleVal': .003
+            }]
+        }]
+    }
+}
+
+
+@mock.patch.object(discovery, 'build')
+def test_per_qubit_noise_from_data(build):
+    service = mock.Mock()
+    build.return_value = service
+    calibrations = service.projects().processors().calibrations()
+    calibrations.list().execute.return_value = ({
+        'calibrations': [_CALIBRATION]
+    })
+    eng = cirq.google.Engine(project_id='myproject')
+    calibration = eng.get_latest_calibration('fake_processor_name')
+    noise_model = simple_noise_from_calibration_metrics(calibration)
```

Comment:
Why are you testing the calibration-getting path as part of testing your method? They seem like entirely separate concerns to me.

If this is the simplest way to create an engine.Calibration instance, we should fix that.

Reply:
Good point. I considered leaving this as an example of how to get calibration data from the engine, but the docstring in the `simple_noise_from_calibration_metrics` function should be sufficient. (Plus, it ended up being 90% mock code and 10% useful examples).

Switched to direct initialization.